base/docs/helpdb/Base.jl

```diff
@@ -784,12 +784,12 @@ Equivalent to `stat(file).mode`
 filemode
 
 """
-    print_joined(io, items, delim, [last])
+    join(io, items, delim, [last])
```

Comment:
by "the line below" do you mean the part outside the docstring? that needs to be valid julia syntax